app/controllers/api/users_controller.rb

```diff
@@ -53,16 +53,32 @@ def destroy
     private
 
     def users
-      sort_key = params.fetch(:sort_key, 'name')
-      order_by = params.fetch(:order_by, 'asc')
-      limit = [params.fetch(:limit, 50).to_i, 1000].min
-      offset = params[:offset]
       @_users ||= User
                     .order(sort_key => order_by)
                     .limit(limit)
                     .offset(offset)
     end
 
+    def sort_key
+      sortable_keys = %w[name id activated_at created_at updated_at]
+      @_sort_key ||= sortable_keys.include?(params[:sort_key]) ? params[:sort_key] : 'name'
+      @_sort_key
```

Comment:
`||=` 代入文だけで返り値はうまく処理できるので、これでよさそう
```suggestion
      @_sort_key ||= sortable_keys.include?(params[:sort_key]) ? params[:sort_key] : 'name'
```